pkg/log/kzerolog/kzerolog.go

```diff
@@ -110,11 +103,14 @@ func truncate(i interface{}, maxLen int) string {
 
 func (ctx *zeroLogContext) logEvent(evt *zerolog.Event, msg string, kvList ...interface{}) {
 	msg = truncate(ctx.prefix, maxLenLogger) + " " + msg
-	evt.Fields(kvList).CallerSkipFrame(2).Msg(msg)
+	evt.Fields(kvList).CallerSkipFrame(4).Msg(msg)
```

Comment:
Trying to understand this, looked at https://pkg.go.dev/github.com/rs/zerolog#Event.CallerSkipFrame but still wondering what does this change ends up doing in our context?

Reply:
Well when using this with lighthouse, a value of 2 displays the file and line # in the logging lib which isn't useful, ie we want the stack frame for our calling code.  Due to changes in the updated logging libs, we now need to skip 4 frames.